spyder/widgets/panels/edgeline.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+This module contains the edge line numebr panel
+"""
+
+from qtpy.QtWidgets import QWidget
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QPainter, QColor
+
+
+class EdgeLine(QWidget):
+    """Source code editor's edge line (default: 79 columns, PEP8)"""
+
+    # --- Qt Overrides
+    # -----------------------------------------------------------------
+    def __init__(self, editor):
+        QWidget.__init__(self, editor)
+        self.code_editor = editor
+        self.column = 79
```

Comment:
Yes please, lets do this, even if we dont we expose the functionality yet.

Basically what you can do is turn the column into a list, but for the moment we only take the first item, so that spyder continues to work just fine ;-)